corporate/views.py

```diff
@@ -285,7 +285,7 @@ def billing_home(request: HttpRequest) -> HttpResponse:
             context.update(
                 plan_name=plan.name,
                 has_active_plan=True,
-                free_trial=free_trial,
+                free_trial=is_realm_on_free_trial(plan.customer.realm),
```

Comment:
If we added a method to `CustomerPlan` - `plan.is_free_trial()` we'd be able to use it here in an idiomatic way (And also as a helper in `is_realm_on_free_trial`) without needing to  `is_realm_on_free_trial(plan.customer.realm)` which is a bit awkward and just goes back to fetching `plan` again. Would that make sense here?

Reply:
Yup. Makes sense. Fixed that.